packages/client/src/views/RegisterForm/ReviewForm.tsx

```diff
@@ -95,6 +95,8 @@ function getEvent(eventType: string) {
       return Event.Birth
     case 'death':
       return Event.Death
+    case 'marriage':
+      return Event.Marriage
```

Comment:
could the eventType be properly typed and handled in some way? Like for example the route definitions wouldn't allow other than strictly typed events, and the user wouldn't flow here at all if the event is something different. Then a `const { event } = useCrvsQueryParams()` or something.. just food for thought for future.

Reply:
Using proper types should prevent us from accidentally pushing a invalid route but there's still the case if a user does so intentionally by changing the URL. I think most of our pages would break that way as we don't really have something like runtime checking in place (e.g. zod)

Reply:
Would need a bit of thought and a refactor; we could look into stuff like this when these components are first refactored to function components. Like a simple

```
useCrvsQueryParams(): {
 const params = useQueryParams();
 
 useEffect(() => {
   if (params?.event not in ['marriage', 'death', 'birth'])
     redirect()
  })
  
  return { event: theStrictlyTypedEvent }
}
```

would be simple to use in any view